tests/test_compute_utils.py

```diff
@@ -373,6 +374,46 @@ def test_surface_averages_shortcut(self):
             err_msg="average without sqrt(g) fail",
         )
 
+    @pytest.mark.unit
+    def test_surface_variance(self):
+        """Test that variance is computed correctly when weights are equal."""
```

Comment:
Do you want to add another test for a case where the weights are not equal?

Reply:
Added. Also moved the tests concerning mean to the surface_average unit test